BLUESPAWN-win-client/src/scan/Detection.cpp

```diff
@@ -21,6 +21,18 @@ size_t ComputeHash(IN CONST std::map<std::wstring, std::wstring>& map) {
     return hash;
 }
 
+size_t ComputeHash(IN CONST std::vector<std::wstring>& values){
+    size_t hash{ 0 };
+
+    std::hash<std::wstring> hasher{};
+    for(auto& val : values){
+        auto first{ hasher(val) };
+        hash = ((hash << 35) | (hash >> 29)) ^ ((first >> 32) | ((first << 32) >> 32));
```

Comment:
does this produce any warnings or is there any overflow concerns on x86 @Jack-McDowell ?

Reply:
Good catch, I'll come up with a fix